### PR TITLE
Dev cleanup

### PR DESCRIPTION
**Pull request by comckay**
removes analyst section, does some cleanup and indenting, removes unnecessary assignment in query, fixes some bugs in query

### CODE REVIEW

File: dashboard/functions/make_plots.R [hunk 1]
```diff
@@ -202,6 +202,44 @@ makeGroupedTimeseries <- function(df, x, y, fill, plot.title = "", xlab = "", yl
...
+  grouped.df$group = grouped.df[[group.col]]
+  grouped.df$total = grouped.df[[data.col]]
+  grouped.df$cumul = grouped.df[[cumul.col]]
+
+  # Reorder factors largest to smallest
+  grouped.df = transform(grouped.df, group = reorder(group, order(total, decreasing = TRUE)))
+
+  plt = ggplot(grouped.df, aes(x = group, y = total)) +
+    geom_bar(stat = "identity", colour = "black") +
+    labs(x = paste0(x.lab), title = title, y = y.lab) +
```
Comment (kmagida110) [inline dashboard/functions/make_plots.R]: Why paste0?

File: dashboard/functions/make_plots.R [hunk 2]
```diff
@@ -202,6 +202,44 @@ makeGroupedTimeseries <- function(df, x, y, fill, plot.title = "", xlab = "", yl
...
+    expand_limits(y = 0) +
+    theme_bw() +
+    theme(axis.text.x = element_text(angle = 90, hjust = 1))
+
+  if (cumul.line) {
+    plt = plt +
+      geom_line(aes(x = group, y = cumul, group = 1), colour = "black") +
+      scale_colour_manual(values = c("Cumulative Graph"))
```
Comment (kmagida110) [inline dashboard/functions/make_plots.R]: You can drop this, it was trying to get a legend, which I decided was not MVP

File: dashboard/get_data/query.R [hunk 3]
```diff
@@ -9,20 +9,24 @@ db.driver <- dbDriver("PostgreSQL")
...
       "title",
-      "FROM", node.table, 
+      "FROM", node.table,
       "WHERE is_disabled = False") %>%
   RPostgreSQL::dbGetQuery(statement = ., conn = db.connection) %>%
-  saveRDS(., "dashboard/data/drupal_node_metadata.RDS")
-  
+  {
+    saveRDS(., "dashboard/data/drupal_node_metadata.RDS")
+    drupal.node.metadata <<- .
```
Comment (kmagida110) [inline dashboard/get_data/query.R]: I thought we weren't doing this since it pollutes the environment

File: dashboard/get_data/query.R [hunk 4]
```diff
@@ -33,62 +37,42 @@ drupal.node.descendants <- paste("SELECT ",
...
-             ) %>%
-             {
-               saveRDS(., "dashboard/data/drupal_node_descendants.RDS")
-               drupal.node.with.descendants <<- .
-             }
+  )
+  ) %>%
+  {
+    saveRDS(., "dashboard/data/drupal_node_descendants.RDS")
+    drupal.node.with.descendants <<- .
```
Comment (kmagida110) [inline dashboard/get_data/query.R]: Same saving df here, do we need to?

File: dashboard/get_data/query.R [hunk 5]
```diff
@@ -98,33 +82,49 @@ ga.conversions <- paste(" SELECT ",
...
-              GROUP BY parent_info.content_type, parent_info.title, d.node_id, d.descendant_id, des_info.content_type,
-              des_info.title"
-
-# Get dataframe and fill na conversions and sessions with 0
-dbGetQuery(db.connection, grouped.sessions.query)  %>% 
-  mutate(conversions = ifelse(is.na(conversions), 0, conversions),
-         sessions = ifelse(is.na(sessions), 0, sessions)) %>%
-    saveRDS(., "dashboard/data/grouped.sessions.conversions.RDS")
\ No newline at end of file
+c("SELECT parent_info.content_type as parent_type,
```
Comment (kmagida110) [inline dashboard/get_data/query.R]: This is fine for now but we need to review this on a overall view as well

**Comment by kmagida110** [comment]
Are you OK with resolving the remaining conflict by removing the analyst dashboard? I assume that is the case but I want to be sure.

File: dashboard/functions/make_plots.R (hunk 1 above)
Comment (comckay) [inline dashboard/functions/make_plots.R]: ... that's your code lol

File: dashboard/functions/make_plots.R (hunk 2 above)
Comment (comckay) [inline dashboard/functions/make_plots.R]: 👍

File: dashboard/get_data/query.R (hunk 3 above)
Comment (comckay) [inline dashboard/get_data/query.R]: this was added back because your queries need it

File: dashboard/get_data/query.R (hunk 4 above)
Comment (comckay) [inline dashboard/get_data/query.R]: this is there because the pieces after rely on this df being in the global env

**Comment by comckay** [comment]
the analyst section is removed in this pr. Is there anything that's leftover?

**Pull request merged**